sdk/src/test/java/com/horizen/account/transaction/EthereumTransactionTest.java

```diff
@@ -44,66 +45,107 @@ public void BeforeEachTest() throws InvalidAlgorithmParameterException, NoSuchAl
+        // Test 5: ethereum transaction object returns transaction type id correctly
         assertEquals((byte) 2, ethereumTransaction.transactionTypeId());
 
-        // Test 5: ethereum transaction object returns version correctly
-        assertEquals(1, ethereumTransaction.version());
+        // Test 6: ethereum transaction object returns version correctly
+        assertEquals(0/*legacy*/, ethereumTransaction.version());
 
-        // Test 6: ethereum transaction instance returns messageToSign correctly
+        // Test 7: ethereum transaction instance returns messageToSign correctly
         EthereumTransaction ethereumTransactionDeserialize = EthereumTransactionSerializer.getSerializer().parseBytes(ethereumTransaction.bytes());
         assert (Arrays.equals(ethereumTransaction.messageToSign(), ethereumTransactionDeserialize.messageToSign()));
 
-        // Test 7: ethereum transaction instance returns Signature correctly
-        assertEquals(null, ethereumTransaction.getSignature());
+        // Test 7: getMaxFeePerGas should be null, as this is a legacy transaction
+        assertNull(ethereumTransaction.getMaxFeePerGas());
+        // Test 8: getMaxPriorityFeePerGas should be null, as this is a legacy transaction
+        assertNull(ethereumTransaction.getMaxPriorityFeePerGas());
+        // Test 9: getChainId should be null, as this is a legacy transaction
+        assertNull(ethereumTransaction.getChainId());
+        // Test 10: getFrom should be null, as this is an unsigned tx
+        assertNull(ethereumTransaction.getFrom());
 
-        // Test 8: ethereum transaction instance returns transaction serializer correctly
+        // Test 11: ethereum transaction instance returns Signature correctly
+        assertNull(ethereumTransaction.getSignature());
+
+        // Test 12: ethereum transaction instance returns transaction serializer correctly
         assertEquals(EthereumTransactionSerializer.getSerializer(), ethereumTransaction.serializer());
 
-        // Test 9: ethereum transaction instance returns null as to is empty or has false size
-        assertEquals(null, ethereumTransaction.getTo());
+        // Test 13: ethereum transaction instance returns null as to is empty or has false size
+        assertNull(ethereumTransaction.getTo());
 
-        // Test 10: ethereum transaction instance returns to proposition address correctly
+        // Test 14: ethereum transaction instance returns to proposition address correctly
         RawTransaction toTx = RawTransaction.createTransaction(someValue,
                 someValue, someValue, "00112233445566778899AABBCCDDEEFF01020304", someValue, "");
         EthereumTransaction toEthereumTransaction = new EthereumTransaction(toTx);
         AddressProposition toProposition = new AddressProposition(BytesUtils.fromHexString("00112233445566778899AABBCCDDEEFF01020304"));
         assert (Arrays.equals(toEthereumTransaction.getTo().address(), toProposition.address()));
```

Comment:
Could use `assertArrayEquals()` everywhere there is an `assert(Arrays.equals(x,y))`.